Project/backend/java-server/app/src/main/java/org/server/UserController.java

```diff
@@ -65,4 +65,21 @@ public ResponseEntity<?> addUser(
             );
         }
     }
+
+    @GetMapping("api/v1/generate_token")
+    public ResponseEntity<String> generateToken() {
+        String alphaNumString = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                + "0123456789"
+                + "abcdefghijklmnopqrstuvxyz";
+        int index;
+        StringBuilder token = new StringBuilder();
+        for (int i = 0; i < 16; i++) {
+            index = (int)(alphaNumString.length() * Math.random());
+            token.append(alphaNumString.charAt(index));
+        }
+//        Object response = new Object() {
+//            public final String userToken = token.toString();
+//        };
+        return ResponseEntity.ok("{\"token\": \"" + token.toString() + "\"}");
```

Comment:
use the json library to create json objects. this prevents us from making errors